### ISSUE TITLE
Unable to use forms inside Reveal

### ISSUE THREAD

**Issue by chuntley** [zurb/foundation-sites-6#281]
When using a form inside of a Reveal modal, you are unable to press the space bar on any input. Actual text input works, but once you press the space bar the modal jumps to the right slightly and throws you out of the input.

To recreate:

```
<p><a data-toggle="exampleModal11">Click me for a modal</a></p>

<div class="reveal" id="exampleModal11" data-reveal>
    <form>
        <label>Test</label>
        <input type="text">
    </form>
</div>
```

**Comment by chuntley** [comment]
Commenting out this line fixes it, but probably has other implications. 

```
Foundation.Keyboard.register('Reveal', {
      'ENTER': 'open',
      //'SPACE': 'open',
      'ESCAPE': 'close',
      'TAB': 'tab_forward',
      'SHIFT_TAB': 'tab_backward'
});
```

**Comment by Owlbertz** [comment]
Obviously it makes sense to not trigger events for ENTER and SPACE while actually inside the modal. I will add a condition so the keyboard listener knows what's going on.

**Comment by zurbchris** [comment]
Fixed with ee92cd8
